### ISSUE TITLE
tmux - Bad CPU type

### ISSUE THREAD

**Issue by hmm-norah** [radare/cydia#3]
Hey!,

I was looking for a build of tmux to use over ssh for the iPhone, and was so happy to come across this repo. However, after installing tmux, this error appears:
```

Norah:~ root# tmux
-sh: /usr/bin/tmux: Bad CPU type in executable
```

From what I've been able to guess, it seems to point to not being built for 64 bit, is this the case?

For reference, I'm running this on an iPhone 8 Plus on the 11.3.1 jailbreak
```
Norah:~ root# uname -a
Darwin Norah 17.5.0 Darwin Kernel Version 17.5.0: Tue Mar 13 21:32:12 PDT 2018; root:xnu-4570.52.2~8/RELEASE_ARM64_T8015 iPhone10,2 arm64 D21AP Darwin
```

If I'm not including something, let me know and I would be happy to add it, thank you for reading this!

**Comment by radare** [comment]
Yes i build tmux when arm32 was a thing. I should update the pkg for both or just arm64. Will do when i have some spare time. Thanks for caring to report

**Comment by radare** [comment]
i have not much spare time, i just pushed some fixes in the pkg, but this is far from complete.

https://github.com/radare/cydia/tree/master/packages/tmux <https://github.com/radare/cydia/tree/master/packages/tmux>

if you have time to finish the patches to get latest tmux to build for iOS send a pullreq and i’ll push the package
